### PR TITLE
auto-generate time meta tag

### PR DESCRIPTION
**Pull request by sukhrajghuman**

### CODE REVIEW

File: site/public/index.html [hunk 1]
```diff
@@ -25,8 +25,8 @@
 		<meta property="og:url" content=" https://www.australia.gov.au" />
 		<meta property="og:site_name" content="Australia.gov.au" />
 		<meta property="og:image" content="homepage-resources/header-logo-agov--dark.png" />
-		<meta property="og:description" content=" Information, advice and services for Australians dealing with Coronavirus (COVID-19)." />
-		<meta property="og:updated_time" content="2020-21-3" />
+    <meta property="og:description" content=" Information, advice and services for Australians dealing with Coronavirus (COVID-19)." />
+    <meta property=“og:updated_time” content="<!--TIME-META-->" />
```
Comment (maxious) [inline site/public/index.html]: property speech marks should be changed back from "smart quotes"

File: meta-date.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+CURRENT_DATE=$(date +‘%d/%m/%Y’)
```
Comment (maxious) [inline meta-date.sh]: open graph datetime suggests iso 8601 so double digits for all dates https://ogp.me/#datetime
(We could also include the time)

Can validate with https://developers.facebook.com/tools/debug/
```
date +"%Y-%m-%dT%H:%M:%S%z"
2020-03-21T20:34:05+1100
```

File: meta-date.sh [hunk 3]
```diff
@@ -1,7 +1,11 @@
+# Generate meta tag for last updated date
 CURRENT_DATE=$(TZ="Australia/Sydney" date +'%Y-%m-%dT%H:%M:%S%z')
-DATE_TEXT=$(TZ="Australia/Sydney" date +'%l:%M%p %Z %A, %d %B %Y')
 META_TAG="<meta property="\"og:updated_time\"" content="\"$CURRENT_DATE\"" />"
-DATE_TAG="<p>$DATE_TEXT<p>"
+
+# Generate HTML tag for last updated date
+DATE_TEXT=$(TZ="Australia/Sydney" date +'%l:%M%p %Z %A, %d %B %Y')
+DATE_TAG="<p>Updated:$DATE_TEXT<p>"
```
Comment (maxious) [inline meta-date.sh]: probably should be a space after the colon there?
```
<p>Updated:12:08PM AEDT Monday, 23 March 2020<p>
```

**Comment by sukhrajghuman** [comment]
@maxious 
I've added a condition whereby the time meta data only gets changed if the commit message begins with `content_update`. This way if we make a non-content change to the `index.html` file (such as an image or script), the time data won't be updated.
The content contributors will need to be made aware to add `content_update` to their commit message though.

**Comment by maxious** [comment]
@sukhrajghuman Shouldn't it be the date of the last commit with that message prefix? Otherwise a date won't be generated every build 😅And I don't want to worry about what merge commits would be called. 

You can search commit messages with the grep option:
```
$ git log --all --grep='update' --format="%ad %s" --date=format:'%l:%M%p %Z %A, %d %B %Y' | head -n1
 6:10pm  Monday, 23 March 2020 update shell script
$ git log --all --grep='update' --format="%ad" --date=format:'%l:%M%p %Z %A, %d %B %Y' | head -n1
 6:10pm  Monday, 23 March 2020
```

**Pull request merged**